csv.inc.php

```diff
@@ -266,155 +196,134 @@ protected function dbInsert($arrIn) // Processes the arrays results using a swit
                     $this->arraySkipped[] = $arrayReturned[1];
                     break;
                 case 'exists':
-                    $this->arrayExist[] = $arrayReturned[1];
+                    $this->arrayExists[] = $arrayReturned[1];
                     break;
                 default:
-                    echo'error on insert';
+                    echo 'error on insert';
             }
         }
-        if ($this->testMode)
-        {
+        if ($this->testMode) {
             $salegroupdb->cancelTransaction();
-        }
-        else
-        {
+        } else {
             $salegroupdb->CommitTransaction();
         }
         return TRUE;
     }
 
-
-    private function print($row) //See source 5
+    protected function setdbConfig($dbConfig) // Preparing database config
     {
-        return "|".$row[0]." |".str_pad($row[1], 15)."|".str_pad($row[2], 38)."|".str_pad($row[3], 2)."|".str_pad($row[4], 8)."|".(($row[5]) ? "      " : "Active")."|".PHP_EOL;
+        if (file_exists($dbConfig) && is_readable($dbConfig)) {
+            require 'dbConfig.php';
+        } else {
+            throw new Exception('No File Found!' . PHP_EOL);
+        }
+        return TRUE;
     }
 
-
-
-    public function get_arrayIn()
+    public function getArrayIn()
 
     {
         return (isset($this->arrayIn)) ? $this->arrayIn : NULL;
     }
 
-
-    public function get_arrayError()
+    public function getArrayError()
 
     {
         return (isset($this->arrayError)) ? $this->arrayError : NULL;
     }
 
-
-    public function get_arrayOut()
+    public function getArrayOut()
     {
         return (isset($this->arrayOut)) ? $this->arrayOut : NULL;
     }
 
-
-    public function get_arraySkipped()
+    public function getArraySkipped()
     {
         return (isset($this->arraySkipped)) ? $this->arraySkipped : NULL;
     }
 
-
-
-    public function get_arrayExist()
+    public function getArrayExists()
     {
-        return (isset($this->arrayExist)) ? $this->arrayExist : NULL;
+        return (isset($this->arrayExists)) ? $this->arrayExists : NULL;
     }
 
-
-    public function get_arrayFail()
+    public function getArrayFail()
     {
         return (isset($this->arrayFail)) ? $this->arrayFail : NULL;
     }
 
-
-    public function get_arraySuccess()
-
+    public function getArraySuccess()
 
 
     {
         return (isset($this->arraySuccess)) ? $this->arraySuccess : NULL;
     }
 
-
-    public function get_num()
+    public function getNum()
 
 
     {
         return $this->num;
 
     }
 
-
-
-
-
-
     public function getOutput()
```

Comment:
Ideally this method would count the max lengths of each column (iterate through the file once at startup) to determine the padding length required.